subprojects/ide-provisioning/build.gradle.kts

```diff
@@ -0,0 +1,29 @@
+plugins {
+    id("profiler.embedded-library")
+}
+
+java {
+    toolchain {
+        languageVersion.set(JavaLanguageVersion.of(17))
+    }
+}
+
+repositories {
+    maven { url = uri("https://www.jetbrains.com/intellij-repository/releases") }
```

Comment:
These repos are subject to cleanup, I hope not all of them are required. It's just copy-paste from JetBrains example repo